Problem1.py

```diff
@@ -0,0 +1,19 @@
+#Time complexity: O(n) where n is the length of input array
```

Comment:
The time complexity will be O(max(nums)) since the freq array is of that length!